src/views/Dashboard.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <div>
+      <b-alert :show="billingInfo.trial_end_days < 20" dismissible style="margin: auto">You have {{ billingInfo.trial_end_days }} days left of your free trial. <b-link to="/billing/plans">Click here to purchase</b-link></b-alert>
```

Comment:
needs to be `billingoInfo && ...` - i'm getting null reference error when i first log in.

Also, `v-if` might be better than `:show`, so the things inside it get ignored completely.

Reply:
Also, other checks is needed - if the days are `0` and `< 0` for "expires today" and "has expired"

Reply:
`v-if` is not applicable here because `:show` is `v-model` under the hood